Makefile

```diff
@@ -0,0 +1,47 @@
+CC := /usr/bin/gcc
+CFLAGS := -O3
+INCL = -I.
+LDFLAGS = -lm -lreadline
+CONFIG = -DHAVE_CONFIG_H 
+DEFS = -DFAST_UART
+SOURCES = elf.c  erc32.c  exec.c  func.c  gr740.c \
```

Comment:
When we have many files and they don't all fit in a single line, it looks nicer if there is one file per line. It is also easier to modify and the diffs look nicer

Reply:
ok